dtfs-central-api/api-tests/v1/gef-facilities/gef-facility-put.api-test.js

```diff
@@ -0,0 +1,122 @@
+const wipeDB = require('../../wipeDB');
+const CONSTANTS = require('../../../src/constants');
+
+const facilitiesCollectionName = 'facilities';
+const dealsCollectionName = 'deals';
+
+const app = require('../../../src/createApp');
+const { as } = require('../../api')(app);
+
+const { APPLICATION } = require('../../mocks/gef/gef-applications');
+
+const testUserCache = require('../../mocks/test-users/api-test-users');
+
+const baseUrl = '/v1/portal/gef/facilities';
+const applicationBaseUrl = '/v1/portal/gef/deals';
+
+const api = require('../../api')(app);
+
+const createDeal = async () => {
+  const { body } = await api.post(APPLICATION).to(applicationBaseUrl);
+  return body;
+};
+
+describe('PUT updateGefFacilities', () => {
+  let aMaker;
+  let mockApplication;
+
+  beforeAll(async () => {
+    const testUsers = await testUserCache.initialise(app);
+    aMaker = testUsers().withRole('maker').one();
+  });
+
+  beforeEach(async () => {
+    await wipeDB.wipe([facilitiesCollectionName]);
+    await wipeDB.wipe([dealsCollectionName]);
```

Comment:
I don't think we need 2 lines here. we can simplify it by writing:
`await wipeDB.wipe(['facilities', 'deals']);`

in this case, you can remove line 4 and 5

Reply:
done

Reply:
@Zainzzkk I can't see the changes reflect here ^

Reply:
done, for some reason it didn't add